docs/man_pages/lib-management/update-kendoui.md

```diff
@@ -1,6 +1,8 @@
 update-kendoui
 ==========
 
+# WARNING: This command is deprecated. Use `$ appbuilder kendoui install` instead
```

Comment:
I think you can safely remove the entire docs for this command and leave just the warning and the Related Commands section. In the related commands section, list only the kendoui commands.